src/scripts/CMSSync/index.ts

```diff
@@ -0,0 +1,20 @@
+import fs from 'fs'
+import api from '../api'
+import settings from './settings'
+import { SettingsObject } from './types'
+import { getFormattedData } from './parser'
+
+const CMSSync = (config: SettingsObject[]) => {
+  config.forEach((item: SettingsObject) => {
+    console.info(` ✅ - ${item.name}`)
```

Comment:
Maybe print the check mark after the file has been created. This output is misleading, it makes you feel that the task has successfully been completed.